cves/kernel/CVE-2013-3223.yml

```diff
@@ -471,9 +555,11 @@ CWE:
 CWE_note: |
   CWE as registered in the NVD. If you are curating, check that this
   is correct and replace this comment with "Manually confirmed".
+
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
   Must be under 30 characters. Optional.
 nickname:
```

Comment:
No cool nickname?

Reply:
Maybe we could call it "[something]bleed" like Heartbleed. /s okay, not every bug needs a name